internal/server/statvar/statvar_hierarchy_util.go

```diff
@@ -61,21 +61,38 @@ func GetRawSvg(ctx context.Context, baseTable *cbt.Table) (
 	return svgResp.StatVarGroups, nil
 }
 
-// BuildParentSvgMap gets the mapping of svg/sv id to the parent svg for that svg/sv.
+// BuildParentSvgMap gets the mapping of svg/sv id to the parent svg for that
+// svg/sv. Only gets the parent svg that have a path to the root node.
 func BuildParentSvgMap(rawSvg map[string]*pb.StatVarGroupNode) map[string][]string {
 	parentSvgMap := map[string][]string{}
```

Comment:
nit: ancestorSvgMap ?

Reply:
This map only holds the direct parent svgs of each sv or svg, so I'm not sure that ancestorSvgMap will make more sense?

Reply:
Ah, I see... sorry i misread it.